### ISSUE TITLE
Doesn't see Android device

### ISSUE THREAD

**Issue by drasnop** [onedoes/snapdrop#77]
Hi there, this looks like an awesome idea but I can't get it to work. Details:

Chrome 67.0.3396.99 on Windows 10
Chrome 67.0.3396.87 on Android 6.0.1 Marshmallow (Nexus 5)
Both devices connected to my home wifi
Server: onedoessnapdrop-server-alonfoqgzq.now.sh (status: up)

I tried refreshing the page multiple times, can't get to see the other device. When I open 2 tabs on the same device, I see the same device showing up, as expected. But I can't get it to see the other device.

No important error message in the console except `GET blob:https://onedoes.github.io/8b33bd4e-36b1-4265-a836-d9bf1809e738` failing.

**Comment by douglasduteil** [comment]
Hum...
They might be a IP rooming problem...
We will need to add more transparency around how the room are created/resolved

**Comment by drasnop** [comment]
Yep, I tried it on a different wifi network (at work) and it worked.

I had the same problem at home with sharedrop.io. They mention that you need to have the same external IP address for the service to work locally; I checked and in my case I had the same external *IPv4* address, but a different *IPv6* address... They are aware of this, but it hasn't been fixed: https://github.com/cowbell/sharedrop/issues/55
